mms/model_service_worker.py

```diff
@@ -298,18 +298,25 @@ def load_model(self, data):
             model_dir = data['modelPath']
             model_name = data['modelName']
             handler = data['handler']
+            entry_point = handler.split(':')
+            if len(entry_point) == 1:
+                entry_point = 'handler'
+                if os.path.exists(os.path.join(model_dir, 'MANIFEST.legacy')):
+                    entry_point = 'inference'
+            elif len(entry_point) == 2:
+                entry_point = entry_point[1]
             batch_size = None
             if 'batchSize' in data:
                 batch_size = int(data['batchSize'])
 
             gpu = None
             if u'gpu' in data:
                 gpu = int(data[u'gpu'])
-
+            self.entry_point = entry_point
             manifest, service_file_path = ModelLoader.load(model_dir, handler)
```

Comment:
ModelLoader should be updated:
1. take entry_point[0] instead of handler
2. Not just load the file, but load the module